### PR TITLE
chore(sdk): bump default API timeouts

### PR DESCRIPTION
**Pull request by annirudh**
Description
-----------
The current default timeouts are too low, especially for the more expensive operations like artifact and sweep creation. This bumps the default from 10s -> 1 minute.

Testing
-------
How was this PR tested?

Checklist
-------
- [ ] Include reference to internal ticket "Fixes WB-NNNN" and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Ensure PR title compliance with the [conventional commits standards](https://github.com/wandb/wandb/blob/main/CONTRIBUTING.md#conventional-commits)

### <samp>🤖 Generated by Copilot at 8b27251</samp>

> _`HTTP` timeout_
> _increased to sixty seconds_
> _autumn of errors_

### CODE REVIEW

**Review by kptkin — APPROVED** [review]

File: wandb/sdk/internal/internal_api.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ class Api:
         Override the settings here.
     """
 
-    HTTP_TIMEOUT = env.get_http_timeout(10)
+    HTTP_TIMEOUT = env.get_http_timeout(60)
```
Comment (kptkin) [inline wandb/sdk/internal/internal_api.py:174]: probably too long

Comment (annirudh) [reply wandb/sdk/internal/internal_api.py:174]: How about 30s? 60s matches the server-side timeout.

Comment (dmitryduev) [reply wandb/sdk/internal/internal_api.py:174]: Yea, let's do 30s

Comment (dmitryduev) [inline wandb/sdk/internal/internal_api.py:174]:
```suggestion
    HTTP_TIMEOUT = env.get_http_timeout(30)
```

**Review by dmitryduev — APPROVED** [review]

File: wandb/apis/public.py [hunk 2]
```diff
@@ -246,7 +246,7 @@ class Api:
             You can also set defaults for `entity`, `project`, and `run`.
     """
 
-    _HTTP_TIMEOUT = env.get_http_timeout(9)
+    _HTTP_TIMEOUT = env.get_http_timeout(60)
```
Comment (dmitryduev) [inline wandb/apis/public.py:249]:
```suggestion
    _HTTP_TIMEOUT = env.get_http_timeout(29)
```

**Pull request merged**